elhouse-cover-sensor-and-alarm/src/main.cpp

```diff
@@ -0,0 +1,92 @@
+#include <Arduino.h>
+
+// defines pins numbers
+const int trigPin1 = 4;
+const int echoPin1 = 5;
+
+const int trigPin2 = 7;
+const int echoPin2 = 8;
+
+const int switchPin = 3;
+const int buzzerPin = 6;
+
+//Distance when cover is on 
+const unsigned int distanceClosed = 130;
+
+// defines variables
+long duration;
+unsigned int distance;
+
+void setup() {
+  // Set up pins for 
+  setupCoverSensor();
+  setupBuzzer();
```

Comment:
I cannot really see the value of splitting the setup of the coversensor and buzzer. They might as well be in the setup() function.